src/server/services/hli/stadspas-router-external-consumer.ts

```diff
@@ -28,6 +28,7 @@ import { StadspasAMSAPPFrontend, StadspasBudget } from './stadspas-types';
 
 const AMSAPP_PROTOCOl = 'amsterdam://';
 const AMSAPP_STADSPAS_DEEP_LINK = `${AMSAPP_PROTOCOl}stadspas`;
+const AMSAPP_LINK_NONCE = 'h70yjZuEZl';
```

Comment:
Moet een nonce niet voor elke page load een nieuwe waarde bevatten? (Dit is iets nieuws wat ik zie, dus had het even opgezocht).

Bron:
>Generating values

> From your web server, generate a random base64-encoded string of at least 128 bits of data from a cryptographically > secure random number generator. Nonces should be generated differently each time the page loads (nonce only once!).
Vanuit: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/nonce

Reply:
Eigenlijk wel.. maar dat wordt niet geverifieerd door de browser. Dit is eigenlijk een "hack" om inline sripts toe te staan.

Reply:
Ahh, dan snap ik het